### PR TITLE
bump kvdb-* versions

### PR DESCRIPTION
**Pull request by insipx**
updates kvdb:
- kvdb: 0.5 -> 0.6
- kvdb-rocksdb: 0.7 -> 0.8
- kvdb-memorydb: 0.5 -> 0.6
- kvdb-web: 0.5 -> 0.6

this version of KVDB updates to librocksdb 6.7.3 and updates to a new kvdb interface

### CODE REVIEW

File: bin/node/bench/src/tempdb.rs [hunk 1]
```diff
@@ -41,17 +41,14 @@ impl KeyValueDB for ParityDbWrapper {
 	}
 
 	/// Write a transaction of changes to the buffer.
-	fn write_buffered(&self, transaction: DBTransaction) {
+	fn write(&self, transaction: DBTransaction) -> io::Result<()> {
 		self.0.commit(
 			transaction.ops.iter().map(|op| match op {
 				kvdb::DBOp::Insert { col, key, value } => (*col as u8, &key[key.len() - 32..], Some(value.to_vec())),
 				kvdb::DBOp::Delete { col, key } => (*col as u8, &key[key.len() - 32..], None),
+				kvdb::DBOp::DeletePrefix { col: _, prefix: _ } => unimplemented!()
```
Comment (bkchr) [inline bin/node/bench/src/tempdb.rs:49]: Why don't we need to implement this?

Comment (insipx) [reply bin/node/bench/src/tempdb.rs:49]: tbh, I'm unfamiliar with parity-db, however i don't think it supports prefix-based retrieval

Comment (bkchr) [reply bin/node/bench/src/tempdb.rs:49]: Ahh that is `ParityDB`, so nevermind ;)

**Comment by insipx** [comment]
> Just to be sure, it would be nice to try to sync Kusama with it.

have been syncing on Kusama for the last 30min at ~50bps with 25 peers, seems to be stable at around there

**Comment by insipx** [comment]
Figured i'd update kvdb in polkadot as well with https://github.com/paritytech/polkadot/pull/1096

> Just to be sure, it would be nice to try to sync Kusama with it.

sync runs stable for me at 50bps/25peers

**Comment by bkchr** [comment]
> Figured i'd update kvdb in polkadot as well with [paritytech/polkadot#1096](https://github.com/paritytech/polkadot/pull/1096)
> 
> > Just to be sure, it would be nice to try to sync Kusama with it.
> 
> sync runs stable for me at 50bps/25peers

I assume the database format did not change?

**Comment by insipx** [comment]
> 
> 
> > Figured i'd update kvdb in polkadot as well with [paritytech/polkadot#1096](https://github.com/paritytech/polkadot/pull/1096)
> > > Just to be sure, it would be nice to try to sync Kusama with it.
> > 
> > 
> > sync runs stable for me at 50bps/25peers
> 
> I assume the database format did not change?

Database format hasn't changed afaik

**Pull request merged**